packages/rx_bloc_cli/bin/compile_bundles.sh

```diff
@@ -34,4 +34,9 @@ dart run mason_cli:mason bundle \
 dart run mason_cli:mason bundle \
   -t dart \
   -o lib/src/templates/ \
-  mason_templates/bricks/lib_auth
\ No newline at end of file
+  mason_templates/bricks/lib_auth
+
+dart run mason_cli:mason bundle \
+  -t dart \
+  -o lib/src/templates/ \
+  mason_templates/bricks/feature_google_login
```

Comment:
Please, edit the folder name to lib_auth_google, so we will have all auth related bricks in a sequence